### DISCUSSION TITLE
My Profile is Missing Reputation Points (I think!)

### DISCUSSION THREAD

**Discussion by killshot13** [dailydotdev/daily#275] [General]
I wasn't sure whether to @ someone on Twitter or submit a bug report, so I figured this would kind of be the middle of the road. 

The Backstory:
At the beginning of this year, I wrote [an original article](https://app.daily.dev/posts/zg2eXEfq1) for daily.dev and it was published in early February on the main blog. 

The Question: 
Based on this timeline, did/does this article meet the eligibility requirements for accruing reputation points?

The Problem:
Assuming that I am interpreting the [eligibility parameters](https://daily.dev/blog/claiming-ownership-on-an-article-you-wrote) correctly and my article was eligible for 1x reputation point for each upvote received, then it appears my account is short 16x reputation points. Unfortunately, I cannot tell exactly where the points were/weren't applied, because the article received a total of 20 upvotes, yet I am only short 16 points, which would indicate some were applied but not others...🤯

So anyway, here is a [screenshot to serve as a point of reference](https://kaplanstudent-my.sharepoint.com/:i:/g/personal/davidrehnert_kaplanstudent_onmicrosoft_com/EQAYkhyeu5xMhBdjAl3hWo4BBvGu93o9Pzn1dIhlgwLQMg?e=4SYs1y). 

And for the record, I don't want to make anyone feel rushed or pressured to find an answer immediately, since I only just now noticed the discrepancy even existed! 😂

Please reach out with any further questions, concerns, or updates. Many thanks!

**Comment by idoshamun** [comment]
Hey @killshot13,

Nice talking to you here as well :)
Could it be that you received the ownership after the article received a few upvotes?
In this case, we usually don't grant reputation retroactively.

**Reply by killshot13** [reply]
@idoshamun 

That may very well have been the case, and I have no reason to suspect otherwise. For this reason, I am glad I did not open an actual issue or file a bug report. 

Thank you for the clarification!

**Reply by idoshamun** [reply]
You bet! Feel free to reach us out at any time in any way :)
